overlord/configstate/configcore/corecfg.go

```diff
@@ -100,5 +100,7 @@ func (cfg plainCoreConfig) GetMaybe(instanceName, key string, result interface{}
 // handlers when configuration is applied to a specific root directory with
 // FilesystemOnlyApply().
 type fsOnlyContext struct {
+	// TODO: this option is meh, maybe something bootloader related?
+	UC20    bool
```

Comment:
this should be something like UC20Recovery ?

Reply:
I went with UC20Recovery here